Sources/Controllers/2.0/RoutePlanning/OASaveTrackViewController.mm

```diff
@@ -542,7 +546,8 @@ - (void) onFolderSelected:(NSString *)selectedFolderName
     _selectedFolderIndex = [_allFolders indexOfObject:selectedFolderName];
     [self updateErrorMessage:_fileName];
     [self generateData];
-    [self.tableView reloadData];
+    [_tableView reloadRowsAtIndexPaths:@[[NSIndexPath indexPathForRow:0 inSection:1]] withRowAnimation:UITableViewRowAnimationNone];
```

Comment:
Please correct this here and in all other places! ReloadRows method and reloadSectionsMethod is specifically designed to be called once for all the necessary index paths/sections.